web-client/src/views/StartCase/CountryUpdated.tsx

```diff
@@ -0,0 +1,103 @@
+import {
+  AddressType,
+  OnBlurHandler,
+  OnChangeCountryTypeHandler,
+} from '@web-client/views/StartCase/AddressUpdated';
+import { COUNTRY_TYPES } from '@shared/business/entities/EntityConstants';
+import { FormGroup } from '../../ustc-ui/FormGroup/FormGroup';
+import { connect } from '@web-client/presenter/shared.cerebral';
+import { state } from '@web-client/presenter/app.cerebral';
+import React from 'react';
+
+type CountryTypes = {
+  addressInfo: AddressType;
+  handleBlur: OnBlurHandler;
+  handleChange: OnChangeCountryTypeHandler;
+  registerRef?: Function;
+  type: string;
+};
+
+const countryDeps = {
+  validationErrors: state.validationErrors,
+};
+
+export const CountryUpdated = connect<CountryTypes, typeof countryDeps>(
+  countryDeps,
+  function Country({
+    addressInfo,
+    handleBlur,
+    handleChange,
+    registerRef,
+    type,
+    validationErrors,
+  }) {
+    return (
+      <React.Fragment>
```

Comment:
Any reason not to nix the `React` import use <>, which we seem to use more elsewhere?